openssl/src/ssl/mod.rs

```diff
@@ -57,6 +57,27 @@
 //!     }
 //! }
 //! ```
+// use bitflags::bitflags;
+// use cfg_if::cfg_if;
+// use foreign_types::{ForeignType, ForeignTypeRef, Opaque};
+use libc::{c_char, c_int, c_long, c_uchar, c_uint, c_ulong, c_void};
+// use once_cell::sync::{Lazy, OnceCell};
+// use std::any::TypeId;
+// use std::cmp;
+// use std::collections::HashMap;
+// use std::ffi::{CStr, CString};
+// use std::fmt;
+// use std::io;
+// use std::io::prelude::*;
+// use std::marker::PhantomData;
+// use std::mem::{self, ManuallyDrop};
+// use std::ops::{Deref, DerefMut};
+// use std::panic::resume_unwind;
+// use std::path::Path;
+// use std::ptr;
+// use std::slice;
+// use std::str;
+// use std::sync::{Arc, Mutex};
```

Comment:
?

Reply:
This was removed in the version we've got in Android, but didn't make it into the merge to produce this patch, I'll remove it.